.github/workflows/_s3_publish.yml

```
@@ -178,14 +178,14 @@ jobs:
                 if [ -n "$TARBALL" ]; then
                   FILENAME=$(basename "$TARBALL")
                   # Create an entry that points to the target-specific path
-                  TARGET_PATH="/versions/$S3_PREFIX/$GIT_VERSION/$FILENAME"
+                  TARGET_PATH="/public/releases/$S3_PREFIX/$GIT_VERSION/$FILENAME"
                   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)